projects/plugins/crm/js/welcome-to-zbs/wizard2.js

```diff
@@ -12,225 +12,209 @@
-		allNextBtn = jQuery( '.nextBtn' ),
-		alBackBtn = jQuery( '.backBtn' ),
-		alListBtn = jQuery( '.stepwizard-step' );
-	allWells.hide();
-	navListItems.on( 'click', function ( e ) {
-		e.preventDefault();
-		var jQuerytarget = jQuery( jQuery( this ).attr( 'href' ) ),
-			jQueryitem = jQuery( this );
-		if ( ! jQueryitem.hasClass( 'disabled' ) ) {
-			navListItems.removeClass( 'btn-primary' ).addClass( 'btn-default' );
-			jQueryitem.addClass( 'btn-primary' );
-			allWells.hide();
-			jQuerytarget.show();
-			jQuerytarget.find( 'input' ).eq( 0 ).focus();
-		}
-	} );
-	alListBtn.on( 'click', function () {
-		zbsJS_welcomeWizard_update_deets();
-	} );
-	allNextBtn.on( 'click', function () {
-		let curStep = jQuery( this ).closest( '.setup-content' ),
-			curStepBtn = curStep.attr( 'id' ),
-			nextStepWizard = jQuery( 'div.setup-panel div a[href="#' + curStepBtn + '"]' )
-				.parent()
-				.next()
-				.children( 'a' ),
-			curInputs = curStep.find( "input[type='text'],input[type='url']" ),
-			isValid = true;
-		jQuery( '.form-group' ).removeClass( 'has-error' );
-		for ( let i = 0; i < curInputs.length; i++ ) {
-			if ( ! curInputs[ i ].validity.valid ) {
-				isValid = false;
-				jQuery( curInputs[ i ] ).closest( '.form-group' ).addClass( 'has-error' );
-			}
-		}
-		if ( isValid ) {
-			nextStepWizard.prop( 'disabled', false ).trigger( 'click' );
-		}
-		//console.log(window.zbsOptions);
-	} );
-	jQuery( 'div.setup-panel div a.btn-primary' ).trigger( 'click' );
-	jQuery( '.zbs-gogogo' )
-		.off( 'click' )
-		.on( 'click', function ( e ) {
-			zbsJS_welcomeWizard_update_deets();
-
-			//console.log("finito with ",zbsOptions);
-			if ( jQuery( this ).hasClass( 'disabled' ) ) {
-				return false;
-			}
-			jQuery( this ).addClass( 'disabled' );
-
-			var t = window.zbsOptions;
-			t.action = 'zbs_wizard_fin';
-			t.security = jQuery( '#zbswf-ajax-nonce' ).val();
-
-			jQuery( '.laststage' ).hide();
-			jQuery( '.finishingupblock' ).show();
-			jQuery( '.finishblock' ).hide();
-
-			i = jQuery.ajax( {
-				url: window.ajaxurl,
-				type: 'POST',
-				data: t,
-				dataType: 'json',
-			} );
-			i.done( function ( msg ) {
-				jQuery( '.laststage' ).hide();
-				jQuery( '.finishingupblock' ).hide();
-				jQuery( '.finishblock' ).show();
-			} );
-			i.fail( function ( msg ) {
-				jQuery( '.laststage' ).hide();
-				jQuery( '.finishingupblock' ).hide();
-				jQuery( '.finishblock' ).show();
-			} );
-		} );
-} );
-/**
- *
- */
-export function zbs_biz_select() {
-	if ( jQuery( '#zbs_crm_type' ).val() == 'Other' ) {
-		jQuery( '#zbs_other_details' ).show();
-		jQuery( '#zbs_other_label' ).removeClass( 'hide' );
-		zbsOptions.zbs_crm_type = jQuery( '#zbs_crm_type' ).val();
-	} else {
-		jQuery( '#zbs_other_details' ).hide();
-		jQuery( '#zbs_other_label' ).addClass( 'hide' );
-		zbsOptions.zbs_crm_type = jQuery( '#zbs_crm_type' ).val();
-	}
-
-	setTimeout( function () {
-		zbs_crm_js_updatePage2();
-	}, 0 );
+    var zbsOptions = {
```

Comment:
I believe the vars here still need to be exported to prevent issues like this one: https://github.com/Automattic/zero-bs-crm/issues/3021#issuecomment-1494253446

The problem would be that the Webpack build steps remove function / variable names when minifying the JS and exporting them is the current solution.

Reply:
Actually, I see now that it wraps those files in a block. Will think on that.